### PR TITLE
add style to leaderboard for PR

### PR DESCRIPTION
**Pull request by supreeti**
Finalize the Leaderboard list app, 
Add the styles needed to make it look great.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -2,28 +2,37 @@
...
+  background-color: rgb(174, 176, 176);
 }
 
 .main {
   display: grid;
-  grid-template-columns: 2fr 2fr;
-  align-items: center;
-  gap: 25px;
+  grid-template-columns: 1fr 1fr;
+  grid-gap: 100px;
```
Comment (ghost) [inline src/style.css]: - [ ] Kindly try to stick to the provided wireframe, especially on your form section

image

### Checkout the provided wireframe

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @supreeti , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights
✅ Correct use of Gitflow
✅ No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (supreeti) [reply src/style.css]: change style as per wireframe.

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED ✔️ 🎉 ✔️ 

Hi @supreeti, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] No linters errors ✔️ 
- [ ] Correct Gitflow ✔️ 

## Optional suggestions

- [OPTIONAL] I noticed that your commit messages are not capitalized, the commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). Don't forget that for your next commit messages 🙃 😁

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**